src/styles/blocks/card.scss

```diff
@@ -0,0 +1,82 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid $card-border-color;
+  background: $white;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    color: $secondary-color;
+    font-size: 10px;
+    font-weight: 400;
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+    align-items: center;
+
+    &_count {
+      color: $main-accent;
+      font-size: 10px;
+      font-weight: 400;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+    align-items: center;
+
+    &_title {
+      color: var(--secondary, $secondary-color);
+      font-size: 12px;
+      font-weight: 400;
+    }
+
+    &_number {
+      color: $main-accent;
+      font-size: 16px;
+      font-weight: 700;
+    }
+  }
+
+  &__btn {
+    display: block;
+    width: 166px;
+    height: 40px;
```

Comment:
it's better not hardcode height for elements with content inside. Use padding instead